docs/sdks/uid2-sdk-ref-csharp-dotnet.md

```diff
@@ -107,60 +102,113 @@ Available information returned through the SDK is outlined in the following tabl
 | `KeysNotSynced` | The client has failed to synchronize keys from the UID2 service. |
 | `VersionNotSupported` |  The client library does not support the version of the encrypted token. |
 
+### Encryption Response Content
+
+When encrypting with the `SharingClient`, the SDK will return the following information:
+
+| Property | Description |
+| :--- | :--- |
+| `Status` | The encryption result status. For a list of possible values and definitions, see [Encryption Response Statuses](#encryption-response-statuses). |
+| `EncryptedData` | The encrypted UID2 advertising token. |
+
+### Encryption Response Statuses
+
+| Value | Description |
+| :--- | :--- |
+| `Success` | The raw UID2 was successfully encrypted and a UID2 advertising token was returned. |
+| `NotAuthorizedForKey` | The requester does not have authorization to use the encryption key. |
+| `NotAuthorizedForMasterKey` | The requester does not have authorization to use the master key. |
+| `NotInitialized` | The client library is waiting to be initialized. |
+| `KeysNotSynced` | The client has failed to synchronize keys from the UID2 service. |
+| `KeyInactive` | The encryption key is not active. |
+| `EncryptionFailure` | A generic encryption failure occurred. |
+<!-- `TokenDecryptFailure` intentionally omitted. Does not seem to be used by SharingClient. -->
+
+## Usage for DSPs
```

Comment:
we should probably move the usages up to the top, references down the bottom (or in a common location)

Reply:
Might leave that for @genwhittTTD to decide.